oak-blob-plugins/src/test/java/org/apache/jackrabbit/oak/plugins/blob/datastore/FSBackendTest.java

```diff
@@ -0,0 +1,252 @@
+package org.apache.jackrabbit.oak.plugins.blob.datastore;
```

Comment:
- Please add Apache license header.
- I think better convert this to IT i.e. add suffix IT. Would require -PintegrationTesting on mvn command line.